_posts/2018-10-01-better-ui-components.md

```diff
@@ -241,20 +240,23 @@ alert[autodismiss] {
     opacity: 0; 
 }
 ```
-Nice! We really got ourselves a useful component without a single build step or polyfill required - imagine that! And check out its friendly little API:
+Nice! We've really got ourselves a useful component without a single build step or polyfill required - imagine that! And check out its friendly little API:
 * `alert` tag
 * `type` attribute (_required_) - one of "success", "warn", or "error"
 * `autodismiss` attribute (_optional_) - if present, the Alert will disappear after four seconds
 * `id`, `class`, `aria-` and other "inherited" attributes still apply
 * `transitionend` event - DOM event, fires after Alert disappears
 * Accepts nested content, including other custom tags
 
-If you didn't know any better you might think this was one of them fancy HTML5 elements. That's a sign we are on the right track! 
+~If you didn't know any better you might think this was one of them fancy HTML5 elements. That's a sign we are on the right track!~
```

Comment:
...sure you want to use the psuedo-hillbilly voice, here??

Reply:
yes, it's my favorite lol